openlibrary/issue_template.md

```diff
@@ -0,0 +1,23 @@
+### Please Describe the problem (or idea)
+
+> What happened just before the problem occurred? Or what problem could this idea solve?
+
+
+> What did you expect to see that you didn't?
+
+
+### Please show us where to look
+
+```https://openlibrary.org/ ...```
+
+
+### Browser, version, and operating system
+
+> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!
+
+
+****
+
+## Thank you!
```

Comment:
We could remove this

Reply:
Just keep a single line spacing too

Reply:
I agree with @tabshaikh, let's please remove the Thank text here and below